reference/interbit-cli/manifest/manifest.js

```diff
@@ -251,9 +241,13 @@
                 "authorizedActions": [
                   "@@MANIFEST/SET_MANIFEST"
                 ]
+              },
+              {
+                alias: 'spoke',
```

Comment:
Malformed JSON (double quotes always, everywhere) and incorrect alias and action type
```json
{
  "alias": "hub",
  "authorizedActions": ["DO_A_SPOKE_THING"]
}
```